tests/nfv/vsperf_installation.pm

```diff
@@ -0,0 +1,43 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: openvswitch installation and CLI test
```

Comment:
you copy/paste summary from openvswitch_dpdk, please update it to what actually happening in this module

Reply:
done